src/metadata_tool/tools/reports.clj

```diff
@@ -66,19 +66,67 @@
                             :body     [{ :type    "text/html; charset=\"UTF-8\""
                                          :content body }] } )))
 
+(defn- activity-stale?
+  [activity stale-date]
+  (let [program-id (:program cfg/config)
+        activity-date (tf/parse (tf/formatters :date) (:contribution-date activity))
+        compare-date (compare stale-date activity-date)]
+    (and (pos? compare-date) (= "INCUBATING" (:state activity)))))
+
 (defn- send-email-to-pmc
   [program-id subject body]
-  (if-not (s/blank? program-id)
-    (send-email (:pmc-mailing-list-address (md/program-metadata program-id))
-                subject
-                body
-                :cc-addresses     program-liaison-email-address
-                :reply-to-address program-liaison-email-address)))
+  ; TODO - enable code below and comment out print commands!
```

Comment:
Unactioned TODO?

Reply:
Not really, this is my very hacky way of testing the code without triggering an email to be sent out. Any suggestion on how to test email reports locally?